src/player.h

```diff
@@ -22,7 +22,11 @@
 #ifndef H_ADPLUG_PLAYER
 #define H_ADPLUG_PLAYER
 
+#if not defined DJGPP
```

Comment:
I see some mentions online about a bug in DJGPP, that you have to ensure that #include <string> happens before any other header files that uses namespaces.

Is this due to this?